### 1. Prepare your workstation

In [1]:
# Install vaderSentiment tool.
!pip install vaderSentiment

# Import the necessary class.
from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer

# Create an object from the class.
sia = SentimentIntensityAnalyzer()

     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 126.0/126.0 kB 3.3 MB/s eta 0:00:00


### 2. Import the data

In [3]:
# Define the input data.
sentences = [
 "Excellent customer service. Also loved the amazing showroom where you can get a real feel of the top quality furniture and get inspiration for room designs.",
 "I found the sales person to be quite rude and stuck up... don't think she realised I know all the suppliers she uses. I got the sofa & chair elsewhere. I will not be coming back.",
 "Thank you for finding replacement crystals for my chandelier.",
 "I have worked with the team in several houses. All work was carried out on time and to budget."]

### 3. Apply sentiment analysis

In [4]:
# Apply sentiment analysis.
for sentence in sentences:
 score = sia.polarity_scores(sentence)['compound']
 print(f'The sentiment value of the sentence :"{sentence}" is : {score}')
 # If you insert this line of code, you will have a line break between the outputs.
 print("="*50)

The sentiment value of the sentence :"Excellent customer service. Also loved the amazing showroom where you can get a real feel of the top quality furniture and get inspiration for room designs." is : 0.9485
The sentiment value of the sentence :"I found the sales person to be quite rude and stuck up... don't think she realised I know all the suppliers she uses. I got the sofa & chair elsewhere. I will not be coming back." is : -0.6764
The sentiment value of the sentence :"Thank you for finding replacement crystals for my chandelier." is : 0.3612
The sentiment value of the sentence :"I have worked with the team in several houses. All work was carried out on time and to budget." is : 0.0


### 4. Calculate the output

In [5]:
# Calculate the percentage of the output.
for sentence in sentences:
 print(f'For the sentence "{sentence}"')
 # Calculate the scores.
 polarity = sia.polarity_scores(sentence)
 pos = polarity['pos']
 neu = polarity['neu']
 neg = polarity['neg']
 
 # Display the scores.
 print(f'The percentage of positive sentiment in :"{sentence}" is : {round(pos*100,2)} %')
 print(f'The percentage of neutral sentiment in :"{sentence}" is : {round(neu*100,2)} %')
 print(f'The percentage of negative sentiment in :"{sentence}" is : {round(neg*100,2)} %')
 print("="*50)

For the sentence "Excellent customer service. Also loved the amazing showroom where you can get a real feel of the top quality furniture and get inspiration for room designs."
The percentage of positive sentiment in :"Excellent customer service. Also loved the amazing showroom where you can get a real feel of the top quality furniture and get inspiration for room designs." is : 44.1 %
The percentage of neutral sentiment in :"Excellent customer service. Also loved the amazing showroom where you can get a real feel of the top quality furniture and get inspiration for room designs." is : 55.9 %
The percentage of negative sentiment in :"Excellent customer service. Also loved the amazing showroom where you can get a real feel of the top quality furniture and get inspiration for room designs." is : 0.0 %
For the sentence "I found the sales person to be quite rude and stuck up... don't think she realised I know all the suppliers she uses. I got the sofa & chair elsewhere. I will not be coming

## Conclusion

Ccontext is very important when you determine whether the output provided by Python is correct. 
In essence, you have to consider the statements and your own subjective opinion

Although Python indicated the sentiment as neutral (55.9%), 
it leaned more towards a positive (44.1%) than a negative (0.0%) sentiment. 
An overall sentiment of 0.9485 is very close to 1. 
If we look at the sentence, it seems like a positive customer review based on the following words: 
excellent, amazing, real feel, and top quality. 